deploy/osd-cluster-ready/50-osd-ready.Job.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+    name: osd-cluster-ready
+    namespace: openshift-monitoring
+spec:
+    activeDeadlineSeconds: 3900 # 1 hour for the silence, 5 minutes for processing/data completion
+    backoffLimit: 0
+    template:
+        metadata:
+            name: osd-cluster-ready
+        spec:
+            containers:
+            - name: osd-cluster-ready
+              image: quay.io/kbater/openshift-cli
```

Comment:
> `quay.io/kbater/openshift-cli`

I think you mean something else, like in an app-sre org?

Reply:
What kind of extra features/tools do we have on top of the default openshift cli image? Is it something like `amtool`?

I understand we didn't have time to come up with a CICD flow to build this one, so this is ok as an MVP for now.